src/Tools/SNBCalc/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("SNBCalc")]
```

Comment:
Pls fill all assembly info files in the solution the same way as in other sn repos.
(title, desc, copyright, version, etc.)